### PR TITLE
Development/imagine rit22

### PR DESCRIPTION
**Pull request by Mster115**
- [ ] No discrepancies across browsers. Eg. chrome vs safari
- [ ] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [ ] CSS (https://jigsaw.w3.org/css-validator/)
- [ ] Google sign-in is operational
- [ ] Accessibility functions
- [ ] Pages can scale without distorting page
- [ ] No dead links
- [ ] Navbar is consistent across the site
- [ ] Pages are screen-reader accessible
- [ ] Contrast meets standards for accessibility
- [ ] Pages are keyboard accessible
- [ ] Passes WAVE Evaluation Tool chrome extension

### CODE REVIEW

File: client/src/components/imagine/TicTacToe/TicTacToeBoard.jsx [hunk 1]
```diff
@@ -29,59 +29,62 @@ const TicTacToeBoard = (props)=> {
...
-                {
-                    gameState.isGameOver === Model.GameState.isGameOver ?// <h2> It's the {activePlayer.name}'s turn </h2>:
-                    <></> : (gameState.isGameOver ? <h2>The {gameState.winner} is the winner</h2>:
-                    <h2> The Game was a Tie</h2>)
-                }
-            </div>
-            <div className="tw-container tw-bg-dark tw-mx-auto tw-space-y-2 md:tw-space-y-0 md:tw-gap-2 md:tw-grid md:tw-grid-cols-3 lg:tw-grid-cols-3 tw-p-3">
-                <div>
+        <>
+        <div className="moduleContainer">
```
Comment (AndreasLc1103) [inline client/src/components/imagine/TicTacToe/TicTacToeBoard.jsx:38]: I like where you went with this more than what I had implemented for functionality in the first place. it makes more sense and gives the user a better prespective.

File: client/src/components/imagine/TicTacToe/TicTacToeBoard.jsx [hunk 2]
```diff
@@ -29,59 +29,62 @@ const TicTacToeBoard = (props)=> {
...
-                        key="start"
-                        >
-                            Continue
-                    </button>
-                }
-            </div>
 
+            </div>
+        </div>
+            {gameState.isGameOver === Model.GameState.isGameOver ?// <h2> It's the {activePlayer.name}'s turn </h2>:
```
Comment (AndreasLc1103) [inline client/src/components/imagine/TicTacToe/TicTacToeBoard.jsx]: remove comments on line here.

File: client/src/components/imagine/components/Bias.js [hunk 3]
```diff
@@ -13,12 +13,14 @@ const Bias = (props) =>{
 
     const throwAlert = ()=>{
         if(biasType==="user"){
-            alert.error("Error: The AI has run into a computational error");
-            alert.error("ErrCode#"+Math.floor(Math.random() * (9999 - 1) + 1)+": "+bias);
+            alert.show("ErrCode#"+Math.floor(Math.random() * (9999 - 1) + 1)+": "+bias,{title:"Error: The AI has run into a computational error based on the appearance of your avatar"})
+            // alert.error("Error: The AI has run into a computational error");
```
Comment (AndreasLc1103) [inline client/src/components/imagine/components/Bias.js]: Remove commented lines/

Comment (AndreasLc1103) [reply client/src/components/imagine/components/Bias.js]: Same thing for the other lines in the same file.

File: client/src/components/imagine/TicTacToe/TicTacToeBoard.jsx [hunk 4]
```diff
@@ -29,59 +29,62 @@ const TicTacToeBoard = (props)=> {
         setGameState({...Model.checkWinner(gameInfo)});
         // swaps players
         setActivePlayer(Model.Players[0]);
+
```
Comment (AndreasLc1103) [inline client/src/components/imagine/TicTacToe/TicTacToeBoard.jsx:32]: remove newline

**Review by AndreasLc1103 — COMMENTED** [review]
Minor fixes

**Review by jajibhee — APPROVED** [review]

**Pull request merged**